### DISCUSSION TITLE
How to install new version of Video downloadhelper in hard disk

### DISCUSSION THREAD

**Discussion by manelcampsvalls** [aclap-dev/video-downloadhelper#23] [Help and Q&A] [answered]
### Your question

   I have Linux Ubuntu 18.04.6 LTS and web browsers Chromium (112.0.5615.49), Mozilla Firefox (113.0.2) and Google Chrome (120.0.6099.224). I downloaded the tar.gz archive and apparently installed it well with Terminal following the instructions (I did not see any error message), but it had not been installed. Do I have to move the tar.gz in a specific folder in hard disk or do I have to enter in a specific folder in the Terminal? I am just a computer user with just some more knowledgements...

**Comment by jcfrog** [comment]
Hi,
did you try the `./vdhcoapp install` after extracting the binaries?
More info here https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized#linux-all

**Comment by manelcampsvalls** [comment]
Right now, after downloading the archive vdhcoapp-2.0.10-linux-aarch64.tar.bz2, and then in the terminal, in the Downloads folder, I have written " # tar xf vdhcoapp-2.0.10-linux-aarch64.tar.bz2 -C ~ " and afterwards " # ~/vdhcoapp-2.0.10/vdhcoapp install ". No message error has appeared and it seemed that vdhcoapp was installed, but when I wanted to use it on the web browser, it did not work.

**Reply by paulrouget** [reply, marked as answer]
Look at the Ubuntu & Linux section here: https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized

```
opt/vdhcoapp/vdhcoapp install # (not as root! Don't use sudo)
sudo apt-get install -y flatpak
flatpak permission-set webextensions net.downloadhelper.coapp snap.firefox yes
```

**Comment by manelcampsvalls** [comment]
I have looked at my opt/ folder and... surprise, there is already a vdhcoapp installed! (look at the screen capture)
![Captura de pantalla de 2024-02-08 18-29-40](https://github.com/aclap-dev/video-downloadhelper/assets/158581752/0c8aeabe-4a1b-4098-bf45-127266e2e388) But neither Google Chrome nor Chromium still do not work.
   Someone before has told me something about the binaries. This noun rings a bell to me, but I do not know what it is. I am just a computer user.

**Reply by paulrouget** [reply]
Download the Deb file and double click on it. That should do it.
If it doesn't work, run the commands I mentioned before.
